src/Entity/Book.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Validator\Constraints as Assert;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\BookRepository")
+ */
+class Book
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=64)
+     * @Assert\Length(max=64)
+     * @Assert\NotBlank
+     * @Assert\Regex("/^[\s\p{L}]+$/u")
```

Comment:
A book title can contain numbers, for example "Rok 1984", and other special chars like ",", ".", "-" etc